inception-active-learning/src/main/resources/META-INF/asciidoc/user-guide/annotation_activeLearning.adoc

```diff
@@ -0,0 +1,47 @@
+[[sect_annotation_recommendation]]
+=== Active Learning
+Once the recommenders are set in the <<sect_projects_recommendation, Project Settings>>, and assuming
```

Comment:
I would start with explaining what active learning actually does. From the paper:

> Active learning is a family of methods which seeks to optimize the learning rate of classification algorithms by intelligently soliciting labels from a human user for which the system only has low confidence. This means that recommenders can make better suggestions with less user interactions, allowing the user to perform quicker and more accurate annotations.